backend/resolver/aws/aws.go

```diff
@@ -136,3 +150,35 @@ func (r *res) Search(ctx context.Context, typeURL, query string, limit uint32) (
 		return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("cannot search for type '%s'", typeURL))
 	}
 }
+
+func (r *res) AutoComplete(ctx context.Context, typeURL, search string) ([]string, error) {
+	if r.topology == nil {
+		return []string{}, fmt.Errorf("to use the autocomplete api you must first setup the topology service")
+	}
+
+	results, _, err := r.topology.Search(ctx, &topologyv1.SearchRequest{
```

Comment:
i think you can probably some of this down into the topology package, but a TODO is good for now. i just don't want us to start copy and pasting without thinking through it when we go to add autocomplete to the next target.